irida_staramr_results/cli.py

```diff
@@ -25,19 +27,25 @@ def init_argparser():
                                  help="This is your IRIDA account username.")
     argument_parser.add_argument("-pw", "--password", action="store",
                                  help="This is your IRIDA account password.")
-    argument_parser.add_argument("-c", "--config", action='store', required=True,
-                                 help='Required. Path to a configuration file. ')
-    argument_parser.add_argument("-a", "--append", action='store_true',
+    argument_parser.add_argument("-c", "--config", action="store", required=True,
+                                 help="Required. Path to a configuration file. ")
+    argument_parser.add_argument("-a", "--append", action="store_true",
                                  help="Append all analysis results to a single output file.")
+    argument_parser.add_argument("--fromDate", action="store",
+                                 help="Download only results of the analysis that were created FROM this date.")
+    argument_parser.add_argument("--toDate", action="store",
```

Comment:
Can we change `--fromDate` to `--from_date` and `--toDate` to `--to_date`. iirc python doesn't use camel case for anything so sticking with snake case is probably better.
Also we should have a shorthand for them too, like `-fd`/`-td` or something.

Reply:
done